threeML/plugins/HAWCLike.py

```diff
@@ -418,80 +418,105 @@ def inner_fit(self):
 
     def display(self, radius=0.5, pulls=False):
 
+        """
+        Plot model&data/residuals vs HAWC analysis bins for all point sources in the model.
+
+        :param radius: Radius of disk around each source over which model/data are evaluated. Default 0.5.
+        :param pulls: Plot pulls ( [excess-model]/model ) rather than fractional difference ( [excess-model]/model )
```

Comment:
I think you have a typo here. The first should be ([excess-model] / uncertainty).